README.md

```diff
@@ -262,6 +262,21 @@ After checking out the repo, run `bin/setup` to install dependencies. Then, run
 `rake spec` to run the tests. You can also run `bin/console` for an interactive
 prompt that will allow you to experiment.
 
+### Git Hooks
```

Comment:
Can you put this section after the existing paragraph ("To install this gem onto your local machine")? since that isn't part of this subheader